integration_test/cases/repo.exs

```diff
@@ -1418,6 +1418,17 @@ defmodule Ecto.Integration.RepoTest do
       assert not_inserted.id == nil
     end
 
+    @tag :returning
+    @tag :with_conflict_target
+    test "on conflict ignore and returning (changeset)" do
+      custom_changeset = Custom.changeset(%Custom{}, %{uuid: Ecto.UUID.generate()})
+      {:ok, inserted} = TestRepo.insert(custom_changeset, on_conflict: :nothing, conflict_target: [:uuid])
+      assert inserted.bid
+
+      {:ok, not_inserted} = TestRepo.insert(custom_changeset, on_conflict: :nothing, conflict_target: [:uuid], returning: true)
+      assert not_inserted.bid == inserted.bid
```

Comment:
I believe this should be true.

Reply:
on_conflcit nothing the database is not going to return anything information. There is nothing we can do about it. You have to do a separate query.

Reply:
The returning true only applies if the operation succeeds.

Reply:
To be super clear, now you can precisely use the fact that the `bid` is not there to know that the operation failed. This means that now you can do a `Repo.get` and get the latest version. :) If bid was there, then the operation suceeded, so you don't need a separate query.

I know this is annoying but we are doing all we can with what the database returns.